manifests/1.2.0/opensearch-dashboards-1.2.0.yml

```diff
@@ -16,4 +16,19 @@ components:
 - name: securityDashboards
   repository: https://github.com/opensearch-project/security-dashboards-plugin.git
   ref: "main"
+- name: queryWorkbenchDashboards
+  repository: https://github.com/opensearch-project/sql.git
+  working_directory: workbench
+  ref: "1.2"
```

Comment:
`ref` refers to the git branch. It will fail if the "1.2" branch doesn't exist so most likely this will be `main` until the repo cuts a `1.2` branch. Also there is a PR to bump this up: https://github.com/opensearch-project/sql/pull/254. This PR will need to be merged before we can add it to the manifest.